source/components/buttons/buttonLongClick/buttonLongClick.ng1.ts

```diff
@@ -86,4 +86,27 @@ let longClickButton: angular.IComponentOptions = buildButton({
 
 angular.module(moduleName, [downgrade.moduleName, promiseModuleName])
 	.component(componentName, longClickButton)
-	.controller(controllerName, LongClickButtonController);
+	.controller(controllerName, LongClickButtonController)
+	.directive('myTouchstart', [function () {
```

Comment:
The `my` prefix makes me a little bit uncomfortable. Nothing worth rejecting over though, since the directive is scoped to the long-click component.